YapDatabase/Extensions/Relationships/YapDatabaseRelationshipTransaction.m

```diff
@@ -2625,17 +2625,29 @@ - (void)preprocessManualEdges:(NSMutableArray *)manualEdges
 				edge->action = YDB_EdgeAction_Delete;
 				edge->flags |= YDB_EdgeFlags_SourceDeleted;
 				edge->flags |= YDB_EdgeFlags_BadSource;
+                if (!(edge->state & YDB_EdgeState_HasEdgeRowid))
+                {
+                    edge->flags |= YDB_EdgeFlags_EdgeNotInDatabase;
+                }
```

Comment:
Throughout this file, it looks like the additions are indented with spaces, but the existing code uses tabs.